components/toggleButton/toggleButton.scss

```diff
@@ -0,0 +1,47 @@
+.ui-toggle-button {
+  cursor: pointer;
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  .ui-toggle-button__item {
+    background-color: var(--tx-generic-color-blank);
+    border: var(--tx-toggle-button-item-border-size) var(--tx-toggle-button-item-border-style) var(--tx-toggle-button-item-border-color);
+    color: var(--tx-generic-color-primary);
+    display: inline-block;
+    font-family: var(--tx-generic-font-secondary-font-family), var(--tx-generic-font-secondary-generic-family);
+    font-size: var(--tx-toggle-button-item-font-size);
+    font-weight: var(--tx-generic-font-secondary-weight-regular);
+    line-height: var(--tx-toggle-button-item-line-height);
+    min-width: 130px;
+    padding: 5px 10px;
+    text-align: center;
+
+    &:first-child {
+      border-bottom-left-radius: var(--tx-toggle-button-item-border-radius);
+      border-right: 0;
+      border-top-left-radius: var(--tx-toggle-button-item-border-radius);
+    }
+
+    &:last-child {
+      border-bottom-right-radius: var(--tx-toggle-button-item-border-radius);
+      border-left: 0;
+      border-top-right-radius: var(--tx-toggle-button-item-border-radius);
+    }
+
+    &_active {
+      background-color: var(--tx-generic-color-active);
+      border-color: var(--tx-generic-color-active);
+      color: var(--tx-generic-color-blank);
+  
+      &::before {
+        color: var(--tx-generic-color-blank);
+        content: '✅';
```

Comment:
Should we use ASCII code instead? It doesn't match design right now.

Reply:
Done.